src/beautifiers/php-cs-fixer.coffee

```diff
@@ -61,7 +61,7 @@ module.exports = class PHPCSFixer extends Beautifier
       options.cs_fixer_config_file = if context? and context.filePath? then @findFile(path.dirname(context.filePath), configFiles)
 
     # Try again to find a config file in the project root
-    if not options.cs_fixer_config_file
+    if not options.cs_fixer_config_file and atom.project.getPaths()[0]
```

Comment:
This impacts other beautifiers, too: https://github.com/Glavin001/atom-beautify/search?l=CoffeeScript&q=atom.project.getPaths&type=Code

I think the `projectPath` should be added to `context` for a reusable API all beautifiers can depend on:
- [ ] Add `projectPath` to `context`: https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/index.coffee#L353-L361
- [ ] Use `context.projectPath`:

```suggestion
    if not options.cs_fixer_config_file and context.projectPath?
```

- [ ] Update other usages to use `context.projectPath`, too: https://github.com/Glavin001/atom-beautify/search?l=CoffeeScript&q=atom.project.getPaths&type=Code